### PR TITLE
Enabled torch.testing._internal.jit_utils.* typechecking.

### PR DESCRIPTION
**Pull request by kauterry**
Fixes #{issue number}

### CODE REVIEW

**Review by malfet — APPROVED** [review]

File: torch/_C/__init__.pyi.in [hunk 1]
```diff
@@ -393,6 +397,8 @@ class AggregationType(Enum):
 
 class FileCheck(object):
     # TODO
+    def check_source_highlighted(self, highlight: str) -> 'FileCheck': ...
+    def run(self, test_string: str) -> None: ...
```
Comment (walterddr) [inline torch/_C/__init__.pyi.in:401]:
```suggestion
    # TODO (add more FileCheck signature)
    def check_source_highlighted(self, highlight: str) -> 'FileCheck': ...
    def run(self, test_string: str) -> None: ...
```

File: torch/_C/__init__.pyi.in [hunk 2]
```diff
@@ -413,6 +419,11 @@ class PyTorchFileWriter(object):
     def write_end_of_file(self) -> None: ...
     ...
 
+def _jit_get_inline_everything_mode() -> _bool: ...
+def _jit_set_inline_everything_mode(enabled: _bool) -> None: ...
+def _jit_pass_dce(Graph) -> None: ...
+def _jit_pass_lint(Graph) -> None: ...
```
Comment (walterddr) [inline torch/_C/__init__.pyi.in:425]: can you move these together with the rest of the `def _jit_*` ?

**Review by walterddr — APPROVED** [review]
overall looks very great!

nit on the ordering of your function signatures.

File: torch/_C/__init__.pyi.in (hunk 1 above)
Comment (kauterry) [reply torch/_C/__init__.pyi.in:401]: Sure, thanks!

File: torch/_C/__init__.pyi.in (hunk 2 above)
Comment (kauterry) [reply torch/_C/__init__.pyi.in:425]: I added them here bcoz they fall under "# Defined in torch/csrc/jit/python/init.cpp".

Comment (walterddr) [reply torch/_C/__init__.pyi.in:425]: good catch. didn't see that since github only shows +/- 5 lines

**Pull request closed without merging**